jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1174,6 +1193,8 @@ public JSONObject getDebugState(boolean full, String endpointId)
     {
         JSONObject debugState = new JSONObject();
         debugState.put("id", id);
+        debugState.put("rtcstatsEnabled", isRtcStatsEnabled);
```

Comment:
Should we include `isCallstatsEnabled`, too?

Reply:
Might be useful, I'll add it